bin/millau/runtime/src/lib.rs

```diff
@@ -1252,3 +1252,49 @@ mod tests {
 		assert!(core::mem::size_of::<RuntimeCall>() <= MAX_CALL_SIZE);
 	}
 }
+
+/// TODO: Note: I know this does not belong here, but I don't want to add it to the
```

Comment:
@svyatonik 
Slava, I am not very proud of this, but this AssethubWestend bridging stuff we dont really need for production-like, but was used for Westend to Millau stuff, wdyt?

Reply:
@bkontur Do whatever you think is better/faster for you now. Don't bother about making Millau<>Rialto code pretty - as you said, we'll remove it soon, so no worries :)

Reply:
ok, cool, so then I am done, Rococo<>Westend asset bridging is working